### PR TITLE
clang-tidy: fixup named argument comments

### PR DESCRIPTION
**Pull request by fanquake**
Fix comments so they are checked/consistent.
Fix incorrect comments.

### CODE REVIEW

File: src/wallet/rpc/backup.cpp [hunk 1]
```diff
@@ -273,19 +273,19 @@ RPCHelpMan importaddress()
 
             pwallet->MarkDirty();
 
-            pwallet->ImportScriptPubKeys(strLabel, {GetScriptForDestination(dest)}, false /* have_solving_data */, true /* apply_label */, 1 /* timestamp */);
+            pwallet->ImportScriptPubKeys(strLabel, {GetScriptForDestination(dest)}, /*have_solving_data=*/false, /*apply_label=*/false, /*timestamp=*/1);
```
Comment (mzumsande) [inline src/wallet/rpc/backup.cpp:276]: `apply_label` is changed from true to false, so the tests fail.

**Comment by aureleoules** [comment]
Should these comments be enforced with `CommentBoolLiterals`, `CommentIntegerLiterals`, `CommentFloatLiterals`, etc.?
https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html

File: src/qt/addresstablemodel.cpp [hunk 2]
```diff
@@ -357,7 +357,7 @@ QString AddressTableModel::addRow(const QString &type, const QString &label, con
         // Check for duplicate addresses
         {
             if (walletModel->wallet().getAddress(
-                    DecodeDestination(strAddress), /* name= */ nullptr, /* is_mine= */ nullptr, /* purpose= */ nullptr))
+                    DecodeDestination(strAddress), /*name=*/nullptr, /*is_mine=*/nullptr, /*purpose=*/nullptr))
```
Comment (MarcoFalke) [inline src/qt/addresstablemodel.cpp:360]: there is nothing wrong with this

**Review by maflcko — COMMENTED** [review]
looks like the only purpose of a bunch of this is to cause needless merge conflicts and backport issues

**Comment by fanquake** [comment]
> looks like the only purpose of a bunch of this is to cause needless merge conflicts and backport issues

The easiest way to know the number of conflicts was to just open a PR, and doing that, might as well "fix" all issues, and see how bad it is. In any case, the change here is now much reduced.

File: src/wallet/wallet.cpp [hunk 3]
```diff
@@ -2381,7 +2381,7 @@ bool CWallet::TopUpKeyPool(unsigned int kpSize)
 util::Result<CTxDestination> CWallet::GetNewDestination(const OutputType type, const std::string label)
 {
     LOCK(cs_wallet);
-    auto spk_man = GetScriptPubKeyMan(type, false /* internal */);
+    auto spk_man = GetScriptPubKeyMan(type, /*internal=*/false);
```
Comment (MarcoFalke) [inline src/wallet/wallet.cpp:2384]: No objection to the changes of this kind, btw

**Review by maflcko — APPROVED** [review]
ACK. This allows clang-tidy to check that the arg name is correct

Unrelated whitespace changes have been dropped

**Comment by hebasto** [comment]
> This allows clang-tidy to check that the arg name is correct

Is it enforced somehow?

**Comment by fanquake** [comment]
> Is it enforced somehow?

clang-tidy & bugprone-argument-comment

**Comment by hebasto** [comment]
> > Is it enforced somehow?
> 
> clang-tidy & bugprone-argument-comment

Why does it not fail currently on the master branch, considering https://github.com/bitcoin/bitcoin/blob/38cbf43dee9203364d429dc2179772eca80d8965/src/.clang-tidy#L14-L15 ?

**Comment by fanquake** [comment]
There are lots of linting related things that are not hard failures on the master branch.

**Comment by MarcoFalke** [comment]
> Why does it not fail currently on the master branch

clang-tidy does not understand the syntax. Fixing the syntax is the goal of this pull

**Comment by hebasto** [comment]
> > Why does it not fail currently on the master branch
> 
> clang-tidy does not understand the syntax. Fixing the syntax is the goal of this pull

Ah, right. I [tested](https://cirrus-ci.com/task/5198591371771904) it a bit, and the `clang-tidy` is able to catch errors like that:
```
/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/src/qt/coincontroldialog.cpp:480:102: error: argument name 'noreason' in comment does not match parameter name 'reason' [bugprone-argument-comment,-warnings-as-errors]
        nPayFee = model->wallet().getMinimumFee(nBytes, m_coin_control, /*returned_target=*/nullptr, /*noreason=*/nullptr);
                                                                                                     ^~~~~~~~~~~~~
                                                                                                     /*reason=*/
./interfaces/wallet.h:248:20: note: 'reason' declared here
        FeeReason* reason) = 0;
--
```
and
```
/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/src/wallet/rpc/backup.cpp:187:97: error: argument name 'timestampmp' in comment does not match parameter name 'timestamp' [bugprone-argument-comment,-warnings-as-errors]
                pwallet->ImportScripts({GetScriptForDestination(WitnessV0KeyHash(vchAddress))}, /*timestampmp=*/0);
                                                                                                ^~~~~~~~~~~~~~~~
                                                                                                /*timestamp=*/
./wallet/wallet.h:602:65: note: 'timestamp' declared here
    bool ImportScripts(const std::set<CScript> scripts, int64_t timestamp) EXCLUSIVE_LOCKS_REQUIRED(cs_wallet);
--
```

Concept ACK.

**Review by hebasto — APPROVED** [review]
ACK 203886c443c4ad76f8a1dba740a286e383e55206, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**